process/TAC-Decision-Process.md

```diff
@@ -0,0 +1,25 @@
+# TAC Decision Process #
+
+The OpenSSF Technical Advisory Council (TAC) is a body comprised of 9 individuals that are either elected for a 2 year term or appointed by the governing board for a 1 year term.  
+
+ !! ADD link to TAC election process !!
+
+The TAC has a chairperson and vice-chairperson that are elected by the TAC members annually that are responsible for organizing and coordinating the TAC to conduct the technical management and oversight of the foundation's Technical Initiatives.  The TAC chair acts as a standing voting member of the Governance Committee (GC), and the vice-chair acting as their alternate/delegate.  A second TAC member, as determined by the TAC, is also included as a voting member of the GC. 
+
+Broadly, there are three main methods that the TAC will use to make decisions and ensure those decisions are appropriately documented:
+
+## 1.) Simple and/or non-consequential changes ##
+For asynchronous work items that do not have long-term/strategic consequences, the TAC will use lazy consensus to determine whether to adopt the proposal and ensure that it will be properly documented.  Issues and PRs should be created within the TAC GitHub repository and assigned to the TAC group for review.  Provided that at least two TAC members agree, support, and approve the suggested change, there are no objections raised by any TAC members, and no less than one week has transpired since the TAC was notified of the change, the proposal will be merged and adopted.
```

Comment:
Can we include an example of what does have long-term/strategic consequences? I think "adoption of a new TI" maybe is long term?

Reply:
we sure can! please feel free to suggest something

Reply:
Isn't that what line 14 is trying to do?